doc/argos-integration.md

```diff
@@ -54,7 +54,7 @@ sudo ldconfig
 
 The Buzz integration library for ARGoS is composed of two elements:
 
-  - A set of ARGoS controllers. At the moment, available controllers include one for the [foot-bot](http://www.swarmanoid.org/swarmanoid_hardware.php.html), one for the Khepera IV robot (both wheeled robots), and one for the [Spiri](http://pleiades.ca|Spiri) (a commercial quad-rotor). More can be added easily by subclassing `CBuzzController`, defined in `$PREFIX/include/buzz/argos/buzz_controller.h`. The variable `$PREFIX` depends on your system and is usually `/usr` or `/usr/local`.
+  - A set of ARGoS controllers. At the moment, available controllers include one for the [foot-bot](http://www.swarmanoid.org/swarmanoid_hardware.php.html), one for the Khepera IV robot (both wheeled robots), and one for the [Spiri](https://github.com/Pleiades-Spiri) (a commercial quad-rotor). More can be added easily by subclassing `CBuzzController`, defined in `$PREFIX/include/buzz/argos/buzz_controller.h`. The variable `$PREFIX` depends on your system and is usually `/usr` or `/usr/local`.
```

Comment:
pleiades.ca is gone but this could either be changed to the github page (above), to  main website (https://pleiadesrobotics.com/) which seems to have some certificate issues currently, or https://spirirobotics.com/.

Reply:
Yes, Pleiades is now Spirirobotics